Animations/Fade/RightBig.js

```diff
@@ -26,5 +28,5 @@ var FadeRight = {
     'transform': 'translateX(2000px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeRightBig', FadeRightBig);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeRightBig', FadeRightBig);
```

Comment:
This does not need to have `'FadeRightBig'` passed into it since we have defined a `className` property in `FadeRightBig`.